### DISCUSSION TITLE
為何無法自訂每頁候選數

### DISCUSSION THREAD

**Discussion by LingqiLeoky** [rime/home#1647] [Q&A] [answered]
我依照定製指南 https://github.com/rime/home/wiki/CustomizationGuide 
對每頁候選數進行定製
![image](https://github.com/user-attachments/assets/d4e5cd85-3874-4419-bd1f-75dd516960c4)
但是
每頁候選數仍然只有五個
並且我再次檢查，確認了該參數的確已經更改
![image](https://github.com/user-attachments/assets/20ba2564-168c-4c01-b2d6-d5d71aa9175b)
但是每頁候選數仍只有5個
求解決辦法

**Comment by LEOYoon-Tsaw** [comment]
不要直接改，用custom打補丁

**Reply by LingqiLeoky** [reply]
額，看圖1，我的確使用了custom打補丁
也確認了補丁成功打上了，但是沒有實際作用

**Reply by fxliang** [reply, marked as answer]
設定每頁候選個數的默認值爲 9，在用戶目錄建立文檔
 default.custom.yaml ：

patch:
  "menu/page_size": 9

**Reply by LingqiLeoky** [reply]
解決了，謝謝您的回覆
